key_commands.rst

```diff
@@ -99,7 +99,7 @@ your local keystore.
 After this you can see if that key was correctly added to your local
 keystore by running ``singularity key list -s`` command:
```

Comment:
All the `singularity` commands throughout this file need to be replaced with `{command}`